File: filmy-app/app/src/main/java/rk/entertainment/filmy/models/configuration/ConfigurationRes.java
package rk.entertainment.filmy.models.configuration;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ConfigurationRes {

    @SerializedName("images")
    private ConfigImageData images;

    @SerializedName("change_keys")
    private List<String> changeKeys = null;

    public ConfigImageData getConfigImageData() {
        return images;
    }

    public void setConfigImageData(ConfigImageData images) {
        this.images = images;
    }

    public List<String> getChangeKeys() {
        return changeKeys;
    }

    public void setChangeKeys(List<String> changeKeys) {
        this.changeKeys = changeKeys;
    }

}

File: filmy-app/app/src/main/java/rk/entertainment/filmy/data/models/configuration/ConfigurationRes.kt
package rk.entertainment.filmy.data.models.configuration

import com.google.gson.annotations.SerializedName

data class ConfigurationRes (
    @SerializedName("images")
    var configImageData: ConfigImageData? = null,

    @SerializedName("change_keys")
    var changeKeys: List<String>? = null
)